### PR TITLE
Fix windows env of SSR example

### PR DESCRIPTION
**Pull request by kaysonwu**
Compilation can not be completed successfully under windows system, it need to use the `cross-env` library

### CODE REVIEW

**Comment by kaysonwu** [comment]
Fix ts-loader cannot find module after compilation PR please see [#538](https://github.com/gregberge/loadable-components/issues/538)

File: examples/server-side-rendering/package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
   },
   "dependencies": {
     "core-js": "^3.0.1",
+    "cross-env": "^7.0.0",
```
Comment (theKashey) [inline examples/server-side-rendering/package.json]: devDependency please.

File: packages/babel-plugin/src/properties/chunkName.js [hunk 2]
```diff
@@ -132,7 +132,6 @@ export default function chunkNameProperty({ types: t }) {
     const values = getExistingChunkNameComment(callPath)
 
     if (!agressiveImport && values) {
-      addOrReplaceChunkNameComment(callPath, values)
```
Comment (theKashey) [inline packages/babel-plugin/src/properties/chunkName.js]: this change should be separated from `crossEnv`, and have a clear test to prove it's necessity  and correctness.

File: examples/server-side-rendering/package.json (hunk 1 above)
Comment (kaysonwu) [reply examples/server-side-rendering/package.json]: The `start` script command also use `NODE_ENV`

File: packages/babel-plugin/src/properties/chunkName.js (hunk 2 above)
Comment (kaysonwu) [reply packages/babel-plugin/src/properties/chunkName.js]: This seems to be github automatic behavior. I have run the tests locally

**Comment by kaysonwu** [comment]
split PR

File: examples/server-side-rendering/package.json (hunk 1 above)
Comment (theKashey) [reply examples/server-side-rendering/package.json]: all scripts are "dev" scripts. Final users are going to use `/dist` + `deps`. Everything not required for the final users should be gone.

**Pull request closed without merging**